File: 3 - Intro to Conditional Statements/Solution.cs
using System.CodeDom.Compiler;
using System.Collections.Generic;
using System.Collections;
using System.ComponentModel;
using System.Diagnostics.CodeAnalysis;
using System.Globalization;
using System.IO;
using System.Linq;
using System.Reflection;
using System.Runtime.Serialization;
using System.Text.RegularExpressions;
using System.Text;
using System;



class Solution
{
    public static void Main(string[] args)
    {
        
        int N = Convert.ToInt32(Console.ReadLine().Trim());
        if (N % 2 != 0) Console.WriteLine("Weird");
        else
        {
            if (N %2==0 && N <= 5) Console.WriteLine("Not Weird");
            else if (N %2==0 && N <= 20) Console.WriteLine("Weird");
            else if (N %2==0 && N > 20) Console.WriteLine("Not Weird");
        }
    }
}

File: 3 - Intro to Conditional Statements/Solution.fs
open System

[<EntryPoint>]
let main argv = 
    let n = Console.ReadLine() |> int
    
    let str = 
        if n % 2 <> 0 then "Weird"
        else if n <= 5 then "Not Weird"
        elif n <= 20 then "Weird"
        else "Not Weird"
    printf "%s" str
    0
